src/app/components/settings/settings.ui

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0" />
+  <requires lib="libadwaita" version="1.0" />
+  <template class="SettingsWindow" parent="AdwPreferencesWindow">
+    <property name="default-width">600</property>
+    <property name="hide-on-close">1</property>
+    <child>
+      <object class="AdwPreferencesPage">
+        <child>
+          <object class="AdwPreferencesGroup">
+            <property name="title" translatable="yes" comments="Header for a group of preference items regarding audio">Audio</property>
+            <child>
+              <object class="AdwComboRow" id="audio_backend">
+                <property name="title" translatable="yes" comments="Title for an item in preferences">Audio Backend</property>
+                <property name="model">
+                  <object class="GtkStringList">
+                    <items>
+                      <item>PulseAudio</item>
+                      <item>ALSA</item>
+                    </items>
+                  </object>
+                </property>
+              </object>
+            </child>
+            <child>
+              <object class="AdwActionRow">
+                <property name="title" translatable="yes" comments="Title for an item in preferences">ALSA Device</property>
+                <property name="subtitle" translatable="yes" comments="Description for the item (ALSA Device) in preferences">Applied only if audio backend is ALSA</property>
+                <child>
+                  <object class="GtkEntry" id="alsa_device"></object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="AdwComboRow" id="player_bitrate">
+                <property name="title" translatable="yes" comments="Title for an item in preferences">Player bitrate</property>
```

Comment:
I think it makes more sense to expose this setting as something like "Audio Quality" with Low/Medium/High options, just like Spotify itself does.

Reply:
Agreed. Current bitrate options (96/160/320 kbps) correspond to [Normal/High/"Very high" in Spotify](https://support.spotify.com/us/article/audio-quality/), so it would be better to reflect that in our preferences?